contrib/ci/build_windows.sh

```diff
@@ -42,6 +42,7 @@ meson setup .. \
     --bindir="bin" \
     -Dbuild=all \
     -Dman=false \
+    -Dplugin_uefi_capsule=disabled \
```

Comment:
shouldn't we be using `disable_auto_if()`?

Reply:
Something like?:

    -if get_option('plugin_uefi_capsule').allowed()
    +if get_option('plugin_uefi_capsule').disable_auto_if(host_machine.system() not in ['linux', 'freebsd']).allowed()

Reply:
That seemed to work, thanks.